### PR TITLE
Do not use `Metadata`

### PR DESCRIPTION
**Pull request by qryxip**
`output_metadata` was unnecessary. I thought `Resolve::extern_crate_name` was private.

### CODE REVIEW

**Comment by qryxip** [comment]
Fixes #16.

File: src/main.rs [hunk 1]
```diff
@@ -387,21 +353,21 @@ fn main() -> Result<(), StrErr> {
 
 	for cmd_info in data.relevant_cmd_infos.iter() {
 		let analysis = cmd_info.get_save_analysis()?;
-		for ext in &analysis.prelude.external_crates {
-			if let Some(dependency_name) = dependency_names_by_lib_true_snakecased_name
-				.get(&cmd_info.pkg)
-				.and_then(|names| names.get(&ext.id.name))
-			{
-				used_dependencies.insert((&cmd_info.pkg.repr, *dependency_name));
+		// may not workspace member
```
Comment (est31) [inline src/main.rs]: may not be*

**Comment by est31** [comment]
Thanks! Will give this a closer look soon.

File: src/main.rs (hunk 1 above)
Comment (qryxip) [reply src/main.rs]: fixed.

**Comment by qryxip** [comment]
Removed `cargo_metadata`.
```
unused dependencies:
`cargo-udeps v0.1.2 (/home/ryo/src/cargo-udeps)`
└─── "cargo_metadata"
```

**Pull request merged**